config/server.yaml

```diff
@@ -11,6 +11,7 @@ server:
   port: 8080
   name: BW3 Server        # name of the BW3 Server instance
   useBroadcast: no        # serve server ip on broadcast request
+  log: false              # enable log file
```

Comment:
docs?

Reply:
Habe ein Paar Sätze zum Logging in der config.md geschrieben. Der logging-Parameter ist jetzt auch in der Server-Konfiguration und funktioniert.